server/init.go

```diff
@@ -26,11 +25,14 @@ func GenerateCoinKey(algo keyring.SignatureAlgo) (sdk.AccAddress, string, error)
 	return sdk.AccAddress(info.GetPubKey().Address()), secret, nil
 }
 
-// GenerateSaveCoinKey returns the address of a public key, along with the secret
-// phrase to recover the private key.
+// Deprecated: GenerateSaveCoinKey generates a new key mnemonic with its addrress.
+// If mnemonic is provided then it's used for key generation.
+// The key is saved in the keyring. The function returns error if overwrite=true and the key
+// already exists.
+// Please use testutils.GenerateSaveCoinKey instead.
 func GenerateSaveCoinKey(
 	keybase keyring.Keyring,
-	keyName, mnemonic string,
+	keyName string,
```

Comment:
we decided to rollback the api breaking, deprecate this function and move it with the mnemonic feature to testutil package.